src/components/BuyModal.re

```diff
@@ -264,11 +264,10 @@ let make = (~animal: Animal.t, ~isExplorer: bool=false) => {
   let goToBuy = RootProvider.useGoToBuy();
   let clearAndPush = RootProvider.useClearNonUrlStateAndPushRoute();
 
-  // TODO:: check if foreclosed!!
   let buttonText =
     switch (currentPriceWei) {
-    | Price(_price) => "Adopt"
-    | Foreclosed => "Claim"
+    | Price(_price) => "Buy"
```

Comment:
Yeah this wording is going to be something that evolves over time I guess as we figure out what works best...